.github/workflows/bindings.yml

```diff
@@ -0,0 +1,102 @@
+name: Bindings
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - '**.md'
+      - 'demo/**'
+  pull_request:
+    branches: [ master ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  android:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        toolchain: [ 1.47.0 ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust ${{ matrix.toolchain }}
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.toolchain }}
+          override: true
+      - name: Install Linux dependencies
+        run: |
+          sudo apt-get install -y libpcre3-dev libssl-dev libzmq3-dev
```

Comment:
I think we also require lib-sqlite3 starting from the next beta — not just here, everywhere

Reply:
I think it's better to add it when it will be necessary (in the PR that updates rgb-node)